trade-finance-manager-ui/server/controllers/helpers/amendments.helper.js

```diff
@@ -86,11 +86,53 @@ const hasAmendmentInProgressDealStage = async (dealId) => {
   return false;
 };
 
+const latestAmendmentValueAccepted = (amendment) => {
+  const { ukefDecision, bankDecision, value, requireUkefApproval } = amendment;
+  const { APPROVED_WITH_CONDITIONS: withConditions } = DECISIONS.UNDERWRITER_MANAGER_DECISIONS;
+  const { APPROVED_WITHOUT_CONDITIONS: withoutConditions } = DECISIONS.UNDERWRITER_MANAGER_DECISIONS;
+  const { PROCEED: proceed } = AMENDMENTS.AMENDMENT_BANK_DECISION;
+
+  const ukefDecisionApproved = ukefDecision?.value === withConditions || ukefDecision?.value === withoutConditions;
+  const bankProceed = bankDecision?.decision === proceed;
+
+  if (!requireUkefApproval && value) {
+    return true;
+  }
+
+  if (value && ukefDecisionApproved && bankProceed) {
+    return true;
+  }
+
+  return false;
+};
+
+const latestAmendmentCoverEndDateAccepted = (amendment) => {
+  const { ukefDecision, bankDecision, coverEndDate, requireUkefApproval } = amendment;
+  const { APPROVED_WITH_CONDITIONS: withConditions } = DECISIONS.UNDERWRITER_MANAGER_DECISIONS;
```

Comment:
we don't need the `withConditions` value here. we can use directly  `APPROVED_WITH_CONDITIONS`

Reply:
done